content/blog/2017/2017-04-05-welcome-to-blue-ocean-editor.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean and the Pipeline Visual Editor"
+tags:
+- blueocean
+- ux
+- pipeline
+- tutorial
+- editor
+
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+Blue Ocean is a new user experience for Jenkins,
+and version 1.0 is now live!
+Blue Ocean makes Jenkins, and continuous delivery, approachable to all members of team.
+In my link:/blog/2017/04/05/welcome-to-blue-ocean[previous post],
+I covered how to install Blue Ocean on your local Jenkins instance and switch to using Blue Ocean.
+As promised, here's a screencast that picks up where that post left off.
+Starting from a clean Jenkins install, the video below will guide you through
+creating and running your first Pipeline in Blue Ocean with the Pipeline Visual Editor.
+
+Please Enjoy!  (And look for my next video soon - where I'll go over the Blue Ocean Pipeline Activity View.)
+
+++++
```

Comment:
I think you might want to put the iframe in `<center>` tags to make sure lines up nicely